WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -361,12 +362,14 @@ extension WooAnalyticsEvent {
             ])
         }
 
-        static func orderStatusChange(flow: Flow, from oldStatus: OrderStatusEnum, to newStatus: OrderStatusEnum) -> WooAnalyticsEvent {
-            WooAnalyticsEvent(statName: .orderStatusChange, properties: [
+        static func orderStatusChange(flow: Flow, orderID: Int64? = nil, from oldStatus: OrderStatusEnum, to newStatus: OrderStatusEnum) -> WooAnalyticsEvent {
```

Comment:
What do you think of not setting the default `nil` value on `orderID` so it has to be explicitly excluded on events that don't require it? (Just a thought; no need to change it.)

Reply:
Make sense, I initially thought that having an ID was the exception but it appears that I'm blind as those IDs totally slipped through me!